odl/contrib/fom/util.py

```diff
@@ -393,3 +394,36 @@ def haarpsi_weight_map(img1, img2, axis):
     np.abs(img2_lvl3, out=img2_lvl3)
 
     return np.maximum(img1_lvl3, img2_lvl3)
+
+
+def radial_sum(image, binning_factor=1):
+    """Sum image values over concentric annuli.
+
+    Parameters
+    ----------
+    image : `DiscreteLp` element
+        Input data whose radial sum should be computed.
+    binning_factor : positive int or float, optional
```

Comment:
an int is a float, so just say float.

Also, the default value should be 1.0 to make this more clear